I've realized that it might be real issue with test and after some investigation I found the reason

```
+++ Running case: verify.staticcheck 
+++ working dir: /home/prow/go/src/k8s.io/kubernetes
+++ command: bash "hack/make-rules/../../hack/verify-staticcheck.sh"
installing staticcheck from vendor
Errors from staticcheck:
test/e2e/network/network_policy.go:104:9: this value of err is never used (SA4006)

Please review the above warnings. You can test via:
  hack/verify-staticcheck.sh <failing package>
If the above warnings do not make sense, you can exempt the line or file. See:
  https://staticcheck.io/docs/#ignoring-problems

+++ exit code: 1
+++ error: 1
[0;31mFAILED[0m   verify-staticcheck.sh	317s
```

--- a/test/e2e/network/network_policy.go
+++ b/test/e2e/network/network_policy.go
@@ -104,12 +104,13 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 			nsB, err := f.CreateNamespace(nsBName, map[string]string{
 				"ns-name": nsBName,
 			})
+			framework.ExpectNoError(err, "Error occurred while creating namespace-b.")
 
 			ginkgo.By("Creating a simple server in another namespace that serves on port 80 and 81.")
-			_, serviceB := createServerPodAndService(f, nsB, "pod-b", []int{80, 81})
+			podB, serviceB := createServerPodAndService(f, nsB, "pod-b", []int{80, 81})
 
 			ginkgo.By("Waiting for pod ready", func() {
-				err := e2epod.WaitTimeoutForPodReadyInNamespace(f.ClientSet, podServer.Name, f.Namespace.Name, framework.PodStartTimeout)
+				err := e2epod.WaitTimeoutForPodReadyInNamespace(f.ClientSet, podB.Name, nsB.Name, framework.PodStartTimeout)
 				framework.ExpectNoError(err)
 			})
 

